README.rst

```diff
@@ -108,19 +109,19 @@ Now you can use ``TiingoClient`` to make your API calls. (Other parameters are a
 
   # Get definitions for fields available in the fundamentals-api, ticker is
   # optional
-  definitions = get_fundamentals_definitions('GOOGL')
+  definitions = client.get_fundamentals_definitions('GOOGL')
 
   # Get fundamentals which require daily-updated (like marketCap). A start-
   # and end-date can be passed. If omited, will get all available data.
-  fundamentals_daily = CLIENT.get_fundamentals_daily('GOOGL',
+  fundamentals_daily = client.get_fundamentals_daily('GOOGL',
```

Comment:
Good catch!

Reply:
You can just add it on a later release. That's no problem, at all.

Reply:
Updated in https://github.com/hydrosquall/tiingo-python/commit/37c82d21a0d7e44b2d1616642e27141d62ab4eb3